versioned_docs/version-8.2/guides/update-guide/connectors/030-to-040.md

```diff
@@ -4,9 +4,6 @@ title: Update 0.3 to 0.4
 description: "Review which adjustments must be made to migrate from Connector SDK 0.3.x to 0.4.0."
 ---
 
-import Tabs from "@theme/Tabs";
-import TabItem from "@theme/TabItem";
-
 <span class="badge badge--primary">Intermediate</span>
```

Comment:
Similar to the Guides content, do you also feel Connectors need a review of skill level moving forward, or you're fairly confident with all of these as-is? Would you like to see time estimates as well (or is this solely something we want to keep within Guides for now?

Reply:
I would entirely remove the skill levels and time estimates from the `/update-guide/` section. We should push back on anything that looks lengthy or challenging, or if we have time to run through the actual guides (including update guides), confirm that they are straightforward.

Reply:
I realize this contradicts what I said in https://github.com/camunda/camunda-platform-docs/pull/2337.

Unless there are significant changes from version to version in an update guide, labeling each of them individually feels noisy.